packages/transactions/src/api.ts

```diff
@@ -46,3 +49,41 @@ export const getListQuery = (): string => gql`
     }
   }
 `;
+
+export const deleteFn = async (transactions: Transaction[]) => {
+  await request(Environment.API_URL, getDeleteMutation(), {
+    transactions,
+  });
+};
+
+export const listQueryFn = async (
+  queryParams: QueryProps<Transaction>
+): Promise<{ data: Transaction[]; totalLength: number }> => {
+  const { first, offset, sortBy } = queryParams;
+
+  const { transactions } = await request(Environment.API_URL, getListQuery(), {
+    first,
+    offset,
+    sort: sortBy?.[0]?.id,
+    desc: !!sortBy?.[0]?.desc,
+  });
+
+  return transactions;
+};
+
+export const detailQueryFn = (id: string) => async (): Promise<Transaction> => {
+  const result = await request(Environment.API_URL, getDetailQuery(), {
+    id,
+  });
+  const { transaction } = result;
+  return transaction;
+};
+
+export const updateTransactionFn = async (
```

Comment:
the function names are inconsistent - in that most don't include `transaction` (presumably because they are in the `/transactions/api`) but the `updateTransactionFn` does. How about renaming to `updateFn`?